tests/test_modeling_bart.py

```diff
@@ -253,9 +253,9 @@ def test_enro_forward(self):
         with torch.no_grad():
             logits, *other_stuff = model(**net_input)
 
-        expected_slice = torch.tensor([9.0078, 10.1113, 14.4787], device=torch_device, dtype=model.dtype)
-        result_slice = logits[0][0][:3]
-        self.assertTrue(torch.allclose(expected_slice, result_slice, atol=TOLERANCE))
+        expected_slice = [9.0078, 10.1113, 14.4787]
+        result_slice = logits[0][0][:3].tolist()
+        self.assertListEqual(expected_slice, result_slice)
```

Comment:
still fails, better traceback.

Reply:
Here we are comparing float numbers for equality -> we should use `torch.allclose(...)` no?